src/lib/FHSS/FHSS.cpp

```diff
@@ -132,7 +132,7 @@ const uint32_t FHSSfreqs[] = {
     FREQ_HZ_TO_REG_VAL(925700000),
     FREQ_HZ_TO_REG_VAL(926300000),
     FREQ_HZ_TO_REG_VAL(926900000)};
-#elif Regulatory_Domain_ISM_2400
+#elif defined Regulatory_Domain_ISM_2400
```

Comment:
parens, like the others

Reply:
FHSS file doesn't use any parentheses for the hopping frequencies so I left it without for the sake of consistencies

Reply:
Ok